### PR TITLE
Add some links, formatting

### PR DESCRIPTION
**Pull request by reergymerej**
I just want to participate!

### CODE REVIEW

File: design_document.md [hunk 1]
```diff
@@ -39,3 +39,6 @@ People should be able to find the website easily and use it as a trailhead to ot
 | Apps | Collection of apps for walkers |
 | Media | Writing, photo essays., etc. |
 | News | About walking related topics |
+| FAQ | Tools to help onboard the curious |
```
Comment (reergymerej) [inline design_document.md:42]: I have lots of questions, like how does one walk without feet.  Honest!

Comment (crainjason) [reply design_document.md:42]: The idea I'm playing with here is that people in wheelchairs, using prosthetics, supported by crutches, etc., may all be interested in "walking" somewhere. Their right to travel should not be infringed, as well.

Where things get murky is where to draw the line. We can't support all people for all things, but what if someone in an electric wheelchair wants to plan a trip across the US? An edge case, but interesting as hell.

**Review by crainjason — APPROVED** [review]

File: design_document.md (hunk 1 above)
Comment (reergymerej) [reply design_document.md:42]: I totally get it.  Those are the "for dummies" questions that would be nice to make visible.

Comment (crainjason) [reply design_document.md:42]: Word.

**Pull request merged**